templates/master/00-master/_base/files/usr-local-bin-etcd-snapshot-backup-sh.yaml

```diff
@@ -45,7 +55,12 @@ contents:
       dl_etcdctl
       backup_etcd_client_certs
       backup_manifest
+      backup_latest_kube_static_resources
       snapshot_data_dir
+      tar rf ${BACKUP_TAR_FILE} -C ${BACKUP_DIR} ${SNAPSHOT_FILE#$BACKUP_DIR}
+      gzip ${BACKUP_TAR_FILE}
+      rm -f ${SNAPSHOT_FILLE}
```

Comment:
`SNAPSHOT_FILLE` typo?

Reply:
Thanks. `rm -f` is silent, and I didn't see the error message.